### PR TITLE
WT-3131 Enable ROTN on Windows

### PR DESCRIPTION
**Pull request by daveh86**

### CODE REVIEW

**Comment by daveh86** [comment]
@keithbostic, can you take a look at this and confirm if its suitable?

**Comment by daveh86** [comment]
retest this please

**Comment by daveh86** [comment]
@ddanderson , can you take a look at this and confirm if its suitable/needed?

File: ext/encryptors/rotn/rotn_encrypt.c [hunk 1]
```diff
@@ -64,6 +64,9 @@
  * In this case, we transform all bytes in the input.
  */
 
+/* Add u_char type for windows compatibility */
+typedef unsigned char   u_char;
+
```
Comment (ddanderson) [inline ext/encryptors/rotn/rotn_encrypt.c]: @daveh86, probably it's better to replace all uses of u_char by uint8_t, which should already be defined (it's in stdint.h).

**Pull request closed without merging**